scrapy/logformatter.py

```diff
@@ -13,25 +13,29 @@
 class LogFormatter(object):
     """Class for generating log messages for different actions.
 
-    All methods must return a dictionary listing the parameters `level`, `msg`
-    and `args` which are going to be used for constructing the log message when
-    calling logging.log.
+    All methods must return a dictionary listing the parameters ``level``, ``msg``
+    and ``args`` which are going to be used for constructing the log message when
+    calling ``logging.log``.
 
     Dictionary keys for the method outputs:
-        * `level` should be the log level for that action, you can use those
-        from the python logging library: logging.DEBUG, logging.INFO,
-        logging.WARNING, logging.ERROR and logging.CRITICAL.
 
-        * `msg` should be a string that can contain different formatting
-        placeholders. This string, formatted with the provided `args`, is going
-        to be the log message for that action.
+        *   ``level`` is the log level for that action, you can use those from the 
+            `python logging library <https://docs.python.org/3/library/logging.html>`_ :
+            ``logging.DEBUG``, ``logging.INFO``, ``logging.WARNING``, ``logging.ERROR``
+            and ``logging.CRITICAL``.
+
+        *   ``msg`` should be a string that can contain different formatting placeholders. This string, formatted
+            with the provided ``args``, is going to be the long message for that action.
+
+        *   ``args`` should be a tuple or dict with the formatting placeholders for ``msg``. The final log message is
+            computed as ``msg % args``.
 
-        * `args` should be a tuple or dict with the formatting placeholders for
-        `msg`.  The final log message is computed as output['msg'] %
-        output['args'].
     """
 
     def crawled(self, request, response, spider):
+        """
+        ``crawled`` is called to log message when the crawler finds a webpage.
```

Comment:
No need to repeat the method name in the doc string.

```suggestion
        Logs a message when the crawler finds a webpage.
```

The same applies to the two methods below as well.